packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/RequirementSubForm/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { CWIconButton } from 'views/components/component_kit/cw_icon_button';
+import { CWSelectList } from 'views/components/component_kit/new_designs/CWSelectList';
+import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
+import {
+  AMOUNT_CONDITIONS,
+  BLOCKCHAINS,
+  SPECIFICATIONS,
+  TOKENS,
+} from '../constants';
+import { RequirementSubFormType } from '../index.types';
+import './index.scss';
+
+const requirementTypes = [
+  { value: TOKENS.COSMOS_TOKEN, label: 'Cosmos base tokens' },
+  { value: SPECIFICATIONS.ERC_20, label: 'ERC-20' },
+  { value: SPECIFICATIONS.ERC_721, label: 'ERC-721' },
+  { value: TOKENS.EVM_TOKEN, label: 'EVM base tokens' },
+];
+const chainTypes = [
+  { value: BLOCKCHAINS.AXIE_INFINITY, label: 'Axie Infinity' },
+  { value: BLOCKCHAINS.COSMOS, label: 'Cosmos' },
+  { value: BLOCKCHAINS.ETHEREUM, label: 'Ethereum' },
+  { value: BLOCKCHAINS.INJECTIVE, label: 'Injective' },
+  { value: BLOCKCHAINS.NEAR, label: 'NEAR' },
+  { value: BLOCKCHAINS.POLKADOT, label: 'Polkadot' },
+  { value: BLOCKCHAINS.POLYGON, label: 'Polygon' },
+  { value: BLOCKCHAINS.SOLANA, label: 'Solana' },
+];
+const conditionTypes = [
+  { value: AMOUNT_CONDITIONS.MORE, label: 'More than' },
+  { value: AMOUNT_CONDITIONS.EQUAL, label: 'Equal to' },
+  { value: AMOUNT_CONDITIONS.LESS, label: 'Less than' },
+];
```

Comment:
you created file with constants, can you move them there?

Reply:
TBH, i don't think this will be "contexful" if we move there, these are not actually constants, but the options objects for the select list

Reply:
makes sense